treemap/treemap.js

```diff
@@ -0,0 +1,81 @@
+import * as d3 from 'd3';
+import gChartcolour from 'g-chartcolour';
+
+
+export function draw() {
+    let rem = 10;
+    let seriesNames = [];
+    const colourScale = d3.scaleOrdinal()
+    // .range(gChartcolour.lineWeb)
+        .domain(seriesNames);
+
+    function chart(parent) {
+        const width = window.plotDim.width;
+        const height = window.plotDim.height;
```

Comment:
I added `window.` to fix the `no-undef` linter warning @sdbernard mentioned. TBTH I don't know why we're always setting plotDim as a global variable via `window` — wouldn't it make sense to scope it privately and access it via getter/setter, much like we do everywhere else? I know the reason is it can sometimes be helpful to have that available, but personally I've never needed it (though I tend to access vars via the debugger).